### ISSUE TITLE
bug: 

### ISSUE THREAD

**Issue by lucasrleandro** [ionic-team/ionic#20402]
The logo icons, on https://ionicons.com/, are all invisible.
When we select the icon, the svg model beside the ion-icon tag is invisible. Also, when we use it in the code, it wont render. 

_zone.js:1127 GET http://localhost:8100/svg/logo-google-playstore-outline.svg 404 (Not Found)_

**Comment by wilk-polarny** [comment]
Have you migrated to a newer Angular version lately?

Take a look at your `angular.json` and search for the `assets` array within `options` array beneath your `build` target.

By default it should look like this:

```
            "assets": [
              {
                "glob": "**/*",
                "input": "src/assets",
                "output": "assets"
              },
              {
                "glob": "**/*.svg",
                "input": "node_modules/ionicons/dist/ionicons/svg",
                "output": "./svg"
              }
            ]
```

**Comment by lucasrleandro** [comment]
I have, it looks like this. 
As I said, the problem also happens on the website. Check it: https://ionicons.com/
Select the logo-apple-outline and you will see its invisible at the preview...

**Comment by wilk-polarny** [comment]
> As I said, the problem also happens on the website. Check it: https://ionicons.com/
Select the logo-apple-outline and you will see its invisible at the preview...

![image](https://user-images.githubusercontent.com/819238/73990446-67bb7480-4949-11ea-9c2d-fbdf5693a5a4.png)

Cannot confirm. They recently upgraded to ionicons 5 - maybe try to clear your browser caches? We had a similar problem when switching from ionicons 3 to 4.

**Comment by lucasrleandro** [comment]
Happens on different browsers. Have you checked it?

**Comment by lucasrleandro** [comment]
Yes, I assume it might be related with the release. Happens in differente browsers, exactly how your print shows

**Comment by lucasrleandro** [comment]
Yes, I assume it might be related with the release. Happens in differente browsers, exactly as your print shows
